lib/flake.nix

```diff
@@ -0,0 +1,97 @@
+{
+  description = "DevOS environment configuriguration library.";
+
+  inputs =
+    {
+      deploy.url = "github:serokell/deploy-rs";
+      deploy.inputs = {
+        utils.follows = "flake-utils";
+      };
+      devshell.url = "github:numtide/devshell";
```

Comment:
I haven't yet made my mind up on whether to place `flk` here or in the template. But since it is a wrapper which might eventually go away, provided that upstream `nix` adapts to nixos use cases more natively, maybe `./lib` is a good place, indeed. More thought needed, though.